processing/src/main/java/org/apache/druid/query/extraction/IdentityExtractionFn.java

```diff
@@ -82,6 +82,12 @@ public boolean equals(Object o)
     return o != null && o instanceof IdentityExtractionFn;
   }
 
+  @Override
+  public int hashCode()
+  {
+    return super.hashCode();
```

Comment:
This isn't correct; it defers to the default implementation of `Object::hashCode`, which will be different for different instances, but the contract of `hashCode` requires that two objects' hash codes be the same if `equals` is true.

It would be better to return a constant.

Reply:
Oops.. Fixed.